pkgs/applications/editors/emacs-modes/melpa-stable-packages.nix

```diff
@@ -22,6 +22,10 @@ self:
 
     super = imported;
 
+    addBuildDepends = pkg: xs: pkg.overrideAttrs (attrs: {
```

Comment:
Why do we need this `addBuildDepends` abstraction instead of just chaining overrideAttrs?

Reply:
Well, personally, I feel like using named abstraction is a better alternative than coding by cut-and-paste---and there are four instances of that cut-and-paste in the file at present.

But if people don't think there's value, then, ehh, whatever.